ecosystem/lava-sdk/src/stateTracker/stateQuery/state_chain_query.ts

```diff
@@ -128,10 +166,16 @@ export class StateChainQuery {
             "Failed fetching current epoch from pairing request."
           );
         }
+
+        let maxCu = pairingResponse.getMaxCu();
+        if (virtualEpoch != undefined) {
+          maxCu += maxCu * virtualEpoch;
```

Comment:
should be virtualEpoch + 1 ?

Reply:
since we are increasing the variable maxCu, so we don't need +1